assets/scss/base/_lists.scss

```diff
@@ -14,6 +25,26 @@ ul, nav ul {
 	}
 }
 
+/*
+Numbered list
+
+An ordered list with numbers where the parent element uses the modifier `.list--numbered` and the child elements use the class `.numbered`
+
+Markup:
```

Comment:
@rpowis We done a search across github and found this is how it was being implemented in the service. @timsb